kong/init.lua

```diff
@@ -685,6 +712,28 @@ function Kong.init()
       error(err)
     end
   end
+
+  if config.request_debug and config.role ~= "control_plane" and is_http_module then
+    local token = config.request_debug_token or utils.uuid()
+
+    local request_debug_token_file = pl_path.join(config.prefix,
+                                                  constants.REQUEST_DEBUG_TOKEN_FILE)
+
+    if pl_path.exists(request_debug_token_file) then
+      local ok, err = pl_file.delete(request_debug_token_file)
+      if not ok then
+        ngx.log(ngx.ERR, "failed to delete old .request_debug_token file: ", err)
+      end
+    end
+
+    local ok, err = pl_file.write(request_debug_token_file, token)
+    if not ok then
+      ngx.log(ngx.ERR, "failed to write .request_debug_token file: ", err)
+    end
+
+    kong.request_debug_token = token
+    ngx.log(ngx.NOTICE, "request_debug_token: ", kong.request_debug_token)
```

Comment:
```suggestion
    ngx.log(ngx.NOTICE, "[request-debug] Token for request debugging: ", kong.request_debug_token)
```

We should use `[request-debug]` for all log output to make filtering easier.

Reply:
done